tests/test_deployer.py

```diff
@@ -96,12 +95,10 @@
 
 
 class TestCreateDeploymentInK8s(object):
-    @pytest.fixture(params=["fiaas", "paasbeta"])
+    @pytest.fixture(params=["fiaas"])
```

Comment:
With just a single value this can be changes to remove the params, the 'request' parameter, and the 'if' and just return the pair of classes directly

Reply:
Addressed in https://github.com/fiaas/mast/pull/26/commits/78410a8b345ed1572bd4ceda1faad10890cd403b